.changelog/2474.txt

```diff
@@ -0,0 +1,11 @@
+```release-note:enhancement
+resource/mongodbatlas_stream_connection: Supports new networking attribute
```

Comment:
nit: would define attribute with backticks and use the full path

Reply:
What do you mean by full path?

Reply:
my bad, thought networking was nested within existing attribute but I see it is root level

Reply:
Okay, I have added the backticks to the attribute